pyi.py

```diff
@@ -695,6 +695,10 @@ def _analyse_union(members: Sequence[ast.expr]) -> UnionAnalysis:
     )
 
 
+_ALLOWED_MATH_ATTRIBUTES_IN_DEFAULTS = frozenset(
+    {"math.inf", "math.e", "math.pi", "math.tau"}
```

Comment:
If your real code uses `numpy.inf`, wouldn't it make sense to add that here as well?

```suggestion
    {"math.inf", "math.e", "math.pi", "math.tau", "numpy.inf"}
```

Reply:
There are some import conventions (see [flake8-import-conventions](https://github.com/joaopalmeiro/flake8-import-conventions)), such as:

```python
import numpy as np
```

in real-world use cases, it should be:

```python
def func(x: float: np.inf) -> None: ...
```

The NumPy constants can be replaced with `math.*`:

```python
In [1]: import numpy as np

In [2]: import math

In [3]: np.inf == math.inf
Out[3]: True

In [4]: np.pi == math.pi
Out[4]: True
```

Since `*.pyi` are separate files other than the source code `*.py`. I think it's OK to only support `math.*` here.